### DISCUSSION TITLE
软件启动与用户帐户控制提示

### DISCUSSION THREAD

**Discussion by bb-DR** [Lunova-Studio/DarkMode2#23] [Q&A]
软件每次启动时都会弹出用户帐户控制的提示，设置软件自启动后，电脑开机就会有该提示。不想在系统设置中完全关闭UAC提示，但也没有找到其他理想的办法，遂前来寻求帮助。

**Comment by bao-cn** [comment]
下个版本修复此功能，请到issues新建问题。
